core/markdown/databricksAccessToken.md

```diff
@@ -0,0 +1,3 @@
+To generate Databricks access token. Please follow the instructions in [here](https://docs.databricks.com/api/latest/authentication.html).
+
+You can access your Databricks workspace [here](https://eastus.azuredatabricks.net/aad/auth?has=&Workspace=/subscriptions/{Outputs.subscriptionId}/resourceGroups/{Outputs.resourceGroupName}/providers/Microsoft.Databricks/workspaces/{Outputs.workspaceName}&WorkspaceResourceGroupUri=/subscriptions/{Outputs.subscriptionId}/resourceGroups/{Outputs.mangedResourceGroupName})
```

Comment:
i need to update the location in the url to the location variable. I will do it now